Formula/gwenhywfar.rb

```diff
@@ -11,18 +11,19 @@ class Gwenhywfar < Formula
     sha256 "a006e0b29c726b480bcdcfc40192d776c7ce2e8e44196122d7aabf3884857c5f" => :sierra
   end
 
-  depends_on "pkg-config" => :build
   depends_on "gettext"
   depends_on "gnutls"
   depends_on "libgcrypt"
   depends_on "openssl@1.1"
+  depends_on "pkg-config"
```

Comment:
Is this a runtime dependency?

Reply:
gwenhywfar-config needs pig-config for execution.